setup.py

```diff
@@ -19,6 +20,16 @@
 from setuptools import setup
 
 
+requires = [
+    'asn1crypto',
+    'click',
+    'construct',
+    'cryptography',
+]
+if sys.version_info[0] == 2:
```

Comment:
Huh, do we actually need to support Python 2 still? If not, I agree that we should deprecate support in a dedicated follow-up.

Reply:
I don't know, to be honest.  I figure it's probably fine to keep it until we have to make a meaningful change here and 2.7 gets in the way?